definitions/ext-rf_scanner/dashboard.json

```diff
@@ -0,0 +1,87 @@
+{
+  "title": "RF Devices",
+  "icon": "line-chart",
+  "grid_column_count": 12,
+  "filter": null,
+  "widgets": [
```

Comment:
This is not a valid dashboard format, but you could import it on an account and export it again to get the new format.